packages/open-api-gateway/README.md

```diff
@@ -100,9 +100,10 @@ new SmithyApiGatewayTsProject({
 This will result in a directory structure similar to the following:
 
 ```
-|_ model/
-    |_ main.smithy - The Smithy model used to define your API. You can define as many .smithy files in here as you like.
-|_ smithy-build/
+|_ smithy/
+    |_ src/
+        |_ main/
+            |_ smithy - The Smithy model used to define your API. You can define as many .smithy files in here as you like.
```

Comment:
We should log the breaking changes and how to migration in the README.

Example:
https://github.com/aws/aws-prototyping-sdk/blob/mainline/packages/nx-monorepo/README.md?plain=1#L5-L6
```markdown
> **BREAKING CHANGES** (pre-release)
> - v0.13.0: `WorkspaceConfig.nxConfig` type `NxConfig => Nx.WorkspaceConfig`, and `overrideProjectTargets` removed in favor of `PDKProject.nx` config to manage Nx project configurations. See [PR 231](https://github.com/aws/aws-prototyping-sdk/pull/231).
```

Reply:
Good call, will update!